client/jetpack-onboarding/steps/summary-completed-steps.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { get, map, without } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import Spinner from 'components/spinner';
+import {
+	getJetpackOnboardingPendingSteps,
```

Comment:
🔤

Reply:
I thought we did default imports before named ones? :thinking:

Reply:
I meant 🔤  for the order of the selectors here.

Reply:
😱 🙀